File: src/entities/Vehicule.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;

/**
 *
 * @author Skander
 */
public class Vehicule {

    
    
    private String marque, modele,couleur,matricule,categorieVoiture;

    public String getCategorieVoiture() {
        return categorieVoiture;
    }

    public void setCategorieVoiture(String categorieVoiture) {
        this.categorieVoiture = categorieVoiture;
    }

    public Vehicule() {
    }

    public Vehicule(String marque, String modele, String couleur, String matricule, String categorieVoiture) {
        this.marque = marque;
        this.modele = modele;
        this.couleur = couleur;
        this.matricule = matricule;
        this.categorieVoiture = categorieVoiture;
    }

    
    
    

    public String getMatricule() {
        return matricule;
    }

    public void setMatricule(String matricule) {
        this.matricule = matricule;
    }

    public String getMarque() {
        return marque;
    }

    public void setMarque(String marque) {
        this.marque = marque;
    }

    public String getModele() {
        return modele;
    }

    public void setModele(String modele) {
        this.modele = modele;
    }

    public String getCouleur() {
        return couleur;
    }

    public void setCouleur(String couleur) {
        this.couleur = couleur;
    }


    @Override
    public String toString() {
        return "Vehicule{" + "marque=" + marque + ", modele=" + modele + ", couleur=" + couleur + ", matricule=" + matricule + ", categorieVoiture=" + categorieVoiture + '}';
    }

    
   
    
   

   

   
    
    
    
    
    
}

File: src/Entity/Vehicule.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Vehicule
 *
 * @ORM\Table(name="vehicule")
 * @ORM\Entity
 */
class Vehicule
{
    /**
     * @var string
     *
     * @ORM\Column(name="matricule", type="string", length=30, nullable=false)
     * @ORM\Id
     */
    private $matricule;

    /**
     * @var string
     *
     * @ORM\Column(name="marque", type="string", length=30, nullable=false)
     */
    private $marque;

    /**
     * @var string
     *
     * @ORM\Column(name="modele", type="string", length=30, nullable=false)
     */
    private $modele;

    /**
     * @var string
     *
     * @ORM\Column(name="couleur", type="string", length=30, nullable=false)
     */
    private $couleur;

    /**
     * @var string
     *
     * @ORM\Column(name="categorieVoiture", type="string", length=30, nullable=false)
     */
    private $categorievoiture;

    public function getMatricule(): ?string
    {
        return $this->matricule;
    }

    public function setMatricule(string $matricule): self
    {
        $this->matricule = $matricule;

        return $this;
    }

    public function getMarque(): ?string
    {
        return $this->marque;
    }

    public function setMarque(string $marque): self
    {
        $this->marque = $marque;

        return $this;
    }

    public function getModele(): ?string
    {
        return $this->modele;
    }

    public function setModele(string $modele): self
    {
        $this->modele = $modele;

        return $this;
    }

    public function getCouleur(): ?string
    {
        return $this->couleur;
    }

    public function setCouleur(string $couleur): self
    {
        $this->couleur = $couleur;

        return $this;
    }

    public function getCategorieVoiture(): ?string
    {
        return $this->categorievoiture;
    }

    public function setCategorieVoiture(string $categorievoiture): self
    {
        $this->categorievoiture = $categorievoiture;

        return $this;
    }
}
